polling_stations/templates/template.html

```diff
@@ -0,0 +1,67 @@
+{% load staticfiles %}<!DOCTYPE html>
+<html lang="en" ng-app>
+  <head>
+    {% block meta %}{% endblock meta %}
+
+    {% block icons %}{% endblock icons %}
+
+    <title>{% block title %}Polling Stations (Alpha){% endblock title %}</title>
```

Comment:
Something tells me the title should be before the meta and icons blocks – I think older IE version have a problem when they don't find the title in the first n bytes of the document…